sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fix LRO for issue: https://github.com/Azure/azure-sdk-for-net/issues/37904
```

Comment:
Feel like unnecessary to add the GitHub issue in change log, we can just describe the issue resolved.

Reply:
Fixed